### DISCUSSION TITLE
Listenner

### DISCUSSION THREAD

**Discussion by AyoubKessemtini** [capacitor-community/bluetooth-le#570] [Q&A] [answered]
How can I listen to arriving data with BleClient?

**Comment by pwespi** [comment, marked as answer]
You can listen to changes of the value of a characteristic by using the [startNotifications](https://github.com/capacitor-community/bluetooth-le#startnotifications) method. See the [usage](https://github.com/capacitor-community/bluetooth-le#usage) section in the README for an example.
